### PR TITLE
Feature theme switch

### PR DESCRIPTION
**Pull request by frozenhelium**
Dependencies

React store: https://github.com/toggle-corp/react-store/pull/564

### CODE REVIEW

File: src/components/general/Navbar/index.js [hunk 1]
```diff
@@ -297,6 +302,7 @@ export default class Navbar extends React.PureComponent {
                 />
                 <Community className={styles.communityDropdown} />
                 <Notifica className={styles.notificationDropdown} />
+                { isDev && <ThemeMenu className={styles.themeMenu} /> }
```
Comment (tnagorra) [inline src/components/general/Navbar/index.js]: Use cloak

File: src/redux/reducers/domainData/userConfig.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+});
+
+const setCurrentTheme = (state, action) => {
+    const { value } = action;
+
+    const settings = {
+        currentThemeId: { $set: value },
+    };
+
+    console.warn('setting theme to', themes[value].title);
```
Comment (tnagorra) [inline src/redux/reducers/domainData/userConfig.js]: Dangling console.warn

File: src/theme/index.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    purpleAndPink: {
+        title: 'Purple & Pink',
+        colors: purpleAndPinkColors,
+        dimens: purpleAndPinkDimens,
+    },
+
+};
+
+export const setTheme = (themeId) => {
+    const theme = themes[themeId];
```
Comment (tnagorra) [inline src/theme/index.js:33]: Can we get deafult theme if there is no theme from specified themeId?

File: src/redux/reducers/domainData/userConfig.js (hunk 2 above)
Comment (frozenhelium) [reply src/redux/reducers/domainData/userConfig.js]: Intended

File: src/theme/index.js (hunk 3 above)
Comment (frozenhelium) [reply src/theme/index.js:33]: Maybe

**Pull request merged**